bigquery/doc.go

```diff
@@ -303,9 +303,10 @@ Errors
 
 Errors returned by this client are often of the type googleapi.Error: https://godoc.org/google.golang.org/api/googleapi#Error
 
-These errors can be introspected for more information by type asserting to the richer *googleapi.Error type. For example:
+These errors can be introspected for more information by using `errors.As` with the richer *googleapi.Error type. For example:
```

Comment:
s/error.As/xerrors.As/ here and below

Reply:
Ah, I wrote this because this is sample user code and for Go 1.13+ they should really use `errors` instead of `xerrrors`... however I guess xerrors works for both, so I can change it.